### ISSUE TITLE
[bug] ctrl + a sometimes not working

### ISSUE THREAD

**Issue by makedir** [JasonStein/Notepads#1068]
selecting everything via ctrl + a is randomly not working, closing the app and reopening seems to fix it.

**Comment by JasonStein** [comment]
I have not seen this before, any context?

**Comment by makedir** [comment]
No. It just randomly stops working. Only fix is to close the app and reopen it. Mostly same issue like https://github.com/JasonStein/Notepads/issues/1071#issuecomment-1093508592

**Comment by JasonStein** [comment]
Have you seen this happen before? Like in previous version of Windows or previous version of the app?

**Comment by makedir** [comment]
No, but I have also not used the app since some time, and wanted to give it a try again. Not sure if this is just since Windows 11 upgrade.

**Comment by makedir** [comment]
Anything new on this when you will fix this? It kinda makes your app useless. I have this issue right now, that in one tab ctrl+a toggles between shift lock back and forth, and switching to another tab in your same app it behaves normally selecting all. How can that even be!?
